fix/expr_data/mpas_2024052700/obs

```diff
@@ -1 +1 @@
-../../.agent/expr_data/mpas_2024052700/obs
```

Comment:
Did the obs for the mpasjedi test case change? Just wondering the reason for this change.

Reply:
Good question. No changes in obs. The new obs is just a link to the old obs. 
```
 obs -> ../mpas_2024052700/obs/
```